dod-advana/gamechanger-web
# Task/uot 113269

## Description
clones hosted in their own pods: 
- clone_meta has been updated; clone_to_advana, clone_to_gamechanger, clone_to_jupiter has been replaced with available_at, an array holding domains
- if 'all' is in available_at, then it. will be available on all domains
- admin clone management panel has a new field to add a comma separated list of domains

## Related Issue/Ticket
https://jira.di2e.net/browse/UOT-113269

## Testing instructions
- restart docker containers; a migration needs to be run. 
- go into postgres and run v101.0_update_clone_meta.sql (null available_at values are changed to 'all')
- go to clone admin panel, edit a clone and add 'localhost' (or whatever domain you're testing on) to available_at
- refresh the page and navigate to that clone. check that it is accessible. 
- edit that clone to not have your domain. refresh, check that it is no longer accessible.

## tests:
ModularGameChangerController
storeCloneMeta
